lab3/kursinis.tex

```diff
@@ -100,15 +100,37 @@ \subsection{Implementation alternatives}
+A link in existing note feature to forward the user to an another third-party quiz app on the internet. This simple implementation alternative requires no implementation whatsoever.  We would create a note with a link to another website, where a new user account would have to be created if the student is using the app for the first time, otherwise logging in is sufficient. Once authenticated the student will be presented with the quiz they wished to take. The duration for implementing this solution is at most 1 day, most of the time being consumed by searching for a suitable third-party app. The price to implement this option will not cost our client anything if they find the suitable third-party app themselves, otherwise, if we are tasked with determining the most suitable third-party app, this implementation will cost a trivial sum. User impact: high. The disadvantage of this approach is that both the teachers and the students will be forced to create a new account in a separate application. This implementation is completely not scalable, meaning we can not add new features or change existing ones, but on the other hand, we do not have to maintain it as it falls upon the owners of the third-party quiz app, saving us money and time as a result. 
```

Comment:
Not just reduction of text, but structurization (as in the note).

a) description, short paragraph describing the alternative (should consider corectness)
b) Duration estimate, estimate of how much time it will take (perhaps with explanatory paragraph)
c) Pricing: how much will it cost (again perhaps with some explanation)
d) User impact (relative, basically “medium”, “high” or “low” or stuff in between). Some text about how users are affected, will they like the feature or not
e) Scalability/maintenance (relative) Text about how easy or hard it will be to maintain such subsystem, add new features and so on.
d) Business risc - )user impact + scalability/maintenance) and pricing ratio, I guess? Or don’t put it if it feels like nonsense.